src/main/java/com/amadeus/Shopping.java

```diff
@@ -116,6 +125,7 @@ public Shopping(Amadeus client) {
     this.flightOffersSearch = new FlightOffersSearch(client);
     this.seatMaps = new SeatMaps(client);
     this.activities = new Activities(client);
+    this.flightAvailabilities = new FlightAvailabilities(client);
```

Comment:
Since the developer will call the API as `amadeus.shopping.availability.flightAvailabilities` this line should be removed and instead: 
- Create an Availability.java class and add in the constructor:
```
this.flightAvailabilities = new FlightAvailabilities(client);
```
- In Shopping.java add in the constructor: 
```
this.availability = new Availability(client);
```
Check the implementation of this [PR](https://github.com/amadeus4dev/amadeus-java/pull/92/files), is very similar.